### PR TITLE
BBD-198: handle redirects

### PR DESCRIPTION
**Pull request by effervescentia**

### CODE REVIEW

**Comment by effervescentia** [comment]
@quincylam1729 @Vic-Dev please review

File: test/unit/handlers.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { handleRedirect } from '../../src/handlers';
+
+describe('handlers', () => {
+  let sandbox;
+  beforeEach(() => sandbox = sinon.sandbox.create());
+  afterEach(() => sandbox.restore());
+
+  it('should attach a handler for redirect events', () => {
+    const redirect = 'my-page.html';
+    sandbox.stub(window.location, 'assign', (url) => expect(url).to.eq(redirect))
```
Comment (quincylam1729) [inline test/unit/handlers.ts]: ;

Comment (quincylam1729) [inline test/unit/handlers.ts]: ;

**Comment by Vic-Dev** [comment]
:)

**Pull request merged**